algoliasearch-core/src/main/java/com/algolia/search/ConfigBase.java

```diff
@@ -82,6 +82,7 @@ public Builder(
     public abstract T getThis();
 
     /** Makes the underlying HTTP Client to use JVM/System settings for the proxy */
+    @Deprecated
```

Comment:
I would add a note in the deprecation attribute telling to use the `builder` instead.

Reply:
`@Deprecated` annotation doesn't support optional attributes before Java 9.

I'll put a comment on the same line instead!

Reply:
I've done this like this before:

https://github.com/algolia/algoliasearch-client-java-2/blob/6289d61dd96bad7dd3c298dd8b4c24e1ab1bc0ce/algoliasearch-core/src/main/java/com/algolia/search/SearchClientPersonalization.java#L19-L27

Reply:
OH, even better, I didn't remember of the javadoc way of doing it. Thanks! Changing it right now.